taxonomies/nutrients.txt

```diff
@@ -1938,49 +1938,96 @@ unit:en: mg
 wikidata:en: Q43656
 wikipedia:en: https://en.wikipedia.org/wiki/Cholesterol
 
+# total carbohydrates include fiber
+zz: Total carbohydrates
```

Comment:
I'd personally prefer the id to be `zz: carbohydrates-total`. What do others think?